modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/js/Tabs.js

```diff
@@ -44,6 +29,26 @@ export default function ({
 	navSpecificationsId,
 }) {
 	const [activeTabKeyValue, setActiveTabKeyValue] = useState(0);
+	const [firstRender, setFirstRender] = useState(true);
+
+	const display = (tabId) => {
+		for (let i = 0; i < tabs.length; i++) {
+			const currentTabId = tabs[i].id;
+			const tabElt = document.getElementById(currentTabId);
```

Comment:
Empty line between variable declaration and usage (`currentTabId`)

Reply:
Plus I would probably name it `tabElement` because we don't want abbreviated names.

Reply:
this one was validated by our SF, maybe we need to review the eslint stuff?

Reply:
Yea we might need to look into that. IIRC I looked into it but discovered it's very tricky to implement.